[jsk_apc2016_common] Build error due to lack of cmake dependencies

I got the following error in `catkin build`
```
Scanning dependencies of target sib_topic_synchronizer
[  6%] Building CXX object CMakeFiles/sib_topic_synchronizer.dir/src/sib_topic_synchronizer.cpp.o
/home/ushun/apc2016/apc_ws/src/start-jsk/jsk_apc/jsk_apc2016_common/src/sib_topic_synchronizer.cpp:5:54: fatal error: jsk_apc2016_common/SegmentationInBinSync.h: 
 #include <jsk_apc2016_common/SegmentationInBinSync.h>
                                                      ^
compilation terminated.
make[2]: *** [CMakeFiles/sib_topic_synchronizer.dir/src/sib_topic_synchronizer.cpp.o]  1
make[1]: *** [CMakeFiles/sib_topic_synchronizer.dir/all]  2
make: *** [all]  2
[jsk_apc2016_common] <== '/home/ushun/apc2016/apc_ws/build/jsk_apc2016_common/build_env.sh /usr/bin/make --jobserver-fds=3,5 -j' failed with return code '2'
                                                                                                                                                 
Failed   <== jsk_apc2016_common         [ 12.0 seconds ]                 
```
What is `SegmentationInBinSync.h`? I couldn't find that file.

--- a/jsk_apc2016_common/CMakeLists.txt
+++ b/jsk_apc2016_common/CMakeLists.txt
@@ -158,8 +158,8 @@ add_executable(cloud_to_spatial_features src/cloud_to_spatial_features.cpp)
 ## Add cmake target dependencies of the executable
 ## same as for the library above
 # add_dependencies(jsk_apc2016_common_node ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
-add_dependencies(sib_topic_synchronizer ${catkin_EXPORTED_TARGETS})
-add_dependencies(cloud_to_spatial_features ${catkin_EXPORTED_TARGETS})
+add_dependencies(sib_topic_synchronizer ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+add_dependencies(cloud_to_spatial_features ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 
 ## Specify libraries to link a library or executable target against
 # target_link_libraries(jsk_apc2016_common_node

